dir.targets

```diff
@@ -43,6 +43,7 @@
     <!-- if building desktop facade, we don't bin place the refs -->
     <BinPlaceRef Condition="'$(BinPlaceRef)' == '' And '$(BuildingDesktopFacade)' != 'true' And ('$(IsReferenceAssembly)' == 'true' OR '$(IsRuntimeAndReferenceAssembly)' == 'true')">true</BinPlaceRef>
     <BinPlaceRuntime Condition="'$(BinPlaceRuntime)' == '' And ('$(IsRuntimeAssembly)' == 'true' OR '$(IsRuntimeAndReferenceAssembly)' == 'true')">true</BinPlaceRuntime>
+    <BinPlaceRuntime Condition="'$(BinPlaceRuntime)' == '' And ('$(IsNETCoreApp)' == 'true' AND '$(IsTestProject)' == 'true' AND '$(BinPlaceNETCoreAppTests)' == 'true')">true</BinPlaceRuntime>
```

Comment:
I tried ``corerun xunit.console.netcore.exe [test dll] [xunit option]``, and it worked for me, or could we collect test directories under tests directory? Flat layout is not mandatory. I just need a set of test suites for netcoreapp.